includes/ucf-degree-api.php

```diff
@@ -101,6 +110,26 @@ public static function get_degree_thumbnail( $object, $field_name, $request ) {
 		}
 
 		/**
+		 * Adds the postmeta to the `meta` field.
+		 * @author Jim Barnes
+		 * @since 1.0.1
+		 * @param $object array | The data object that will be returned to the Rest API
+		 * @param $field_name string | The field name
+		 * @param $request array | The request object, which includes get and post parameters
+		 * @return mixed | In this case an array of postmeta.
+		 **/
+		public static function get_post_meta( $object, $field_name, $request ) {
+			$retval = array();
+			$postmeta = get_post_meta( $object['id'] );
+			
+			foreach( $postmeta as $key => $val ) {
+				$retval[$key] = $val[0];
```

Comment:
Updated to only include `degree_` meta_keys and renamed the field `degree_meta`.